alert-common/src/main/java/com/synopsys/integration/alert/common/channel/message/ChannelMessageParser.java

```diff
@@ -134,10 +133,6 @@ public String createFooter(MessageContentGroup messageContentGroup) {
 
     protected abstract String getLineSeparator();
 
-    protected String getListItemPrefix() {
```

Comment:
The reason this exists is to be overridable in different channel implementations. SonarCloud doesn't know that.

Reply:
I think we may want to add this method back if we ever need to override it. Currently there is nowhere in Alert that overrides the method.